legal-api/src/legal_api/models/alternate_name.py

```diff
@@ -333,8 +333,7 @@ def _slim_json(self):
                         if self.business_start_date
                         else None
                     ),
-                    "nameType": self.name_type.name,
-                    "operatingName": self.name,  # will be removed in the future
+                    "nameType": self.name_type.name
```

Comment:
Just a comment not a big issue, this may trigger linting failure without comma, but why does it not have any CI check?

Reply:
Good catch! I might be because when I created my PR, I had main set as the base by mistake. After creating it, I changed the base branch so maybe that prevented the CI from triggering.